pyflakes/test/test_doctests.py

```diff
@@ -367,12 +367,7 @@ class TestOther(_DoctestMixin, TestOther):
 
 
 class TestImports(_DoctestMixin, TestImports):
-
-    def test_futureImport(self):
-        """XXX This test can't work in a doctest"""
-
-    def test_futureImportUsed(self):
-        """XXX This test can't work in a doctest"""
+    pass
```

Comment:
Sure; I'll update this patch after https://github.com/pyflakes/pyflakes/pull/42 is merged, copying the docstring that is merged for that test case.

Anything else?